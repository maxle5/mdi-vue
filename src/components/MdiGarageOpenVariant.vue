
<template>
  <svg :width="size" :height="size" viewBox="0 0 24 24">
      <path d="M22 9V20H20V11H4V20H2V9L12 5L22 9M19 12H5V14H19V12Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiGarageOpenVariant",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
  })
</script>
