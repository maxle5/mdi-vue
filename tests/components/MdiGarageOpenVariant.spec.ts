
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiGarageOpenVariant from "../../src/components/MdiGarageOpenVariant.vue";

test("MdiGarageOpenVariant snapshot", () => {
  const wrapper = mount(MdiGarageOpenVariant, {});
  expect(wrapper.html()).toMatchSnapshot();
});
