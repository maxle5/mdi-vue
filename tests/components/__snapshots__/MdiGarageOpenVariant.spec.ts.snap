// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiGarageOpenVariant snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M22 9V20H20V11H4V20H2V9L12 5L22 9M19 12H5V14H19V12Z" style="fill: currentColor;"></path>
</svg>"
`;
